cachito/web/api_v1.py

```diff
@@ -238,12 +238,7 @@ def create_request():
             tasks.fetch_npm_source.si(request.id, npm_package_configs).on_error(error_callback)
         )
 
-    chain_tasks.extend(
-        [
-            tasks.create_bundle_archive.si(request.id).on_error(error_callback),
-            tasks.set_request_state.si(request.id, "complete", "Completed successfully"),
-        ]
-    )
+    chain_tasks.extend([tasks.create_bundle_archive.si(request.id).on_error(error_callback)])
```

Comment:
Could you please use append here instead of extend since you are only adding a single element?

Reply:
ok, I'll submit new PR for that

Reply:
https://github.com/release-engineering/cachito/pull/235